openvino_wrapper_lib/src/models/base_model.cpp

```diff
@@ -89,3 +90,54 @@ Models::ObjectDetectionModel::ObjectDetectionModel(
   const std::string& model_loc,
   int max_batch_size)
 : BaseModel(label_loc, model_loc, max_batch_size) {}
+
+cv::Mat Models::ObjectDetectionModel::resizeImage(const cv::Mat &orig_image, const int resized_width, const int resized_height,
```

Comment:
this func has too many parameters.
Can we make an opti, e.g.:
- encapsulate dx and dy in cv::Point or cv::Point2f
- encapsulate border_dx in a structure